enterprise/server/cmd/ci_runner/main.go

```diff
@@ -676,6 +676,17 @@ func run() error {
 			return status.WrapError(err, "failed to extract bazelisk")
 		}
 		*bazelCommand = bazeliskPath
+
+		if _, err := runBashCommand(ctx, fmt.Sprintf("sudo ln -s %s /usr/local/bin/bazel", *bazelCommand), nil, "", io.Discard); err != nil {
+			if !strings.Contains(err.Error(), "File exists") {
+				return status.InternalErrorf("could not create a symlink for bazel: %s", err.Error())
+			}
+		}
+		if _, err := runBashCommand(ctx, fmt.Sprintf("sudo ln -s %s /usr/local/bin/bazelisk", *bazelCommand), nil, "", io.Discard); err != nil {
+			if !strings.Contains(err.Error(), "File exists") {
+				return status.InternalErrorf("could not create a symlink for bazel: %s", err.Error())
+			}
+		}
```

Comment:
instead of symlinking to /usr/local/bin I think it'd be better to add this dir to PATH (and symlink bazel to bazelisk)

```
// Symlink bazel to bazelisk
if err := os.Symlink(bazeliskBinaryName, filepath.Join(rootDir, bazelBinaryName)); err != nil {
  // ...
}
// Update PATH to include the root dir
os.Setenv("PATH", rootDir + ":" + os.Getenv("PATH"))
```